Your PR cannot be built. See the log part
```
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:3.0.1:compile (default-compile) on project org.jkiss.dbeaver.net.ssh.sshj: Compilation failure: Compilation failure: 
[ERROR] /var/jenkins_home/jobs/DBeaverDesktop/jobs/dbeaver-ce-github-checks/branches/PR-19857/workspace/dbeaver/plugins/org.jkiss.dbeaver.net.ssh.sshj/src/org/jkiss/dbeaver/model/net/ssh/SSHImplementationSshj.java:[95] 
[ERROR] 	prevClient.getHostname(), prevClient.getPort()));
[ERROR] 	           ^^^^^^^^^^^
[ERROR] The method getHostname() is undefined for the type SSHClient
[ERROR] /var/jenkins_home/jobs/DBeaverDesktop/jobs/dbeaver-ce-github-checks/branches/PR-19857/workspace/dbeaver/plugins/org.jkiss.dbeaver.net.ssh.sshj/src/org/jkiss/dbeaver/model/net/ssh/SSHImplementationSshj.java:[95] 
[ERROR] 	prevClient.getHostname(), prevClient.getPort()));
[ERROR] 	                                     ^^^^^^^
[ERROR] The method getPort() is undefined for the type SSHClient
[ERROR] 2 problems (2 errors)
[ERROR] -> [Help 1]
```

Which version of sshj do you use locally?

--- a/plugins/org.jkiss.dbeaver.net.ssh.sshj/src/org/jkiss/dbeaver/model/net/ssh/SSHImplementationSshj.java
+++ b/plugins/org.jkiss.dbeaver.net.ssh.sshj/src/org/jkiss/dbeaver/model/net/ssh/SSHImplementationSshj.java
@@ -18,6 +18,7 @@ package org.jkiss.dbeaver.model.net.ssh;
 
 import com.jcraft.jsch.Identity;
 import net.schmizz.sshj.SSHClient;
+import net.schmizz.sshj.connection.channel.direct.DirectConnection;
 import net.schmizz.sshj.connection.channel.direct.LocalPortForwarder;
 import net.schmizz.sshj.connection.channel.direct.Parameters;
 import net.schmizz.sshj.sftp.SFTPClient;
@@ -86,14 +87,17 @@ public class SSHImplementationSshj extends SSHImplementationAbstract {
 
             try {
                 if (index > 0) {
-                    final int port = setPortForwarding(clients[index - 1], host.getHostname(), host.getPort());
+                    final SSHClient prevClient = clients[index - 1];
 
                     monitor.subTask(String.format(
-                        "Instantiate tunnel %s:%d -> %s:%d",
-                        hosts[index - 1].getHostname(), port,
-                        host.getHostname(), host.getPort()));
+                        "Instantiate tunnel to %s:%d via %s:%d",
+                        host.getHostname(), host.getPort(),
+                        prevClient.getRemoteHostname(), prevClient.getRemotePort()));
 
-                    client.connect("localhost", port);
+                    final DirectConnection tunnel = prevClient.newDirectConnection(
+                        host.getHostname(), host.getPort()
+                    );
+                    client.connectVia(tunnel);
                 } else {
                     monitor.subTask(String.format(
                         "Instantiate tunnel to %s:%d",

